src/components/templates/Preplaya/VenuePreview.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { Venue } from "types/Venue";
+import "./VenuePreview.scss";
+import { BURN_VENUE_TEMPLATES } from "settings";
+import UserList from "components/molecules/UserList";
+import { useSelector } from "hooks/useSelector";
+import { venueInsideUrl } from "utils/url";
+import { WithId } from "utils/id";
+
+interface VenuePreviewProps {
+  venue: WithId<Venue>;
+}
+
+const VenuePreview: React.FC<VenuePreviewProps> = ({ venue }) => {
+  const { partygoers } = useSelector((state) => ({
+    partygoers: state.firestore.ordered.partygoers,
+  }));
```

Comment:
`const partGoers = useSelector(staet => state.firestore.ordered.partgoers`

avoids rerenders when other stuff changes in redux